gravitee-am-ui/src/app/domain/settings/users/user/profile/profile.component.ts

```diff
@@ -127,9 +127,7 @@ export class UserProfileComponent implements OnInit {
       .subscribe(res => {
         if (res) {
           this.userService.unlock(this.domainId, this.user.id).subscribe(() => {
-            this.user.accountNonLocked = true;
-            this.user.accountLockedAt = null;
-            this.user.accountLockedUntil = null;
+            this.unlockUser(this.user);
```

Comment:
can't we do this.user.unlockUser() and remove private unlockUser(user) function?

Reply:
It's not possible (there is no type User) but you are right I should have something like that